coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -0,0 +1,301 @@
+      </v-container>
+
+      <v-container id="submit-container" class="pt-0">
+        <div class="filing-buttons">
+          <v-btn
+            id="cod-pay-btn"
+            color="primary"
+            large
+            @click="submit">
+            File &amp; Pay
+          </v-btn>
+          <v-btn
+            id="cod-cancel-btn"
+            large
+            to="/dashboard">
+            Cancel
+          </v-btn>
+        </div>
+      </v-container>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import axios from '@/axios-auth'
+import Directors from '@/components/AnnualReport/Directors.vue'
+import { Affix } from 'vue-affix'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import { mapState, mapActions, mapGetters } from 'vuex'
+
+export default {
+  name: 'StandaloneDirectorsFiling',
+
+  components: {
+    Directors,
+    SbcFeeSummary,
+    Affix
+  },
+
+  data () {
+    return {
+      showLoading: false,
+      loadingMsg: 'Redirecting to PayBC to Process Your Payment',
+      directorsChange: false,
+      filingData: [],
+      dialog: false,
+      lastFilingDate: 'something' // TODO
+    }
+  },
+
+  computed: {
+    ...mapState(['noAGM', 'regOffAddrChange',
+      'validated', 'currentDate', 'corpNum',
+      'entityName', 'entityIncNo', 'entityFoundingDate', 'currentARStatus',
+      'addressesFormValid', 'directorFormValid'])
+
+  },
+
+  mounted () {
+    // if tombstone data isn't set, redirect to home
+    if (!this.corpNum) {
+      this.$router.push('/')
+    } else {
+      // load initial data
+      // TODO - anything here?
+    }
+  },
+
+  methods: {
+    ...mapActions(['setDirectorFormValid', 'setValidated']),
+
+    directorsChangeEventHandler (val) {
+      this.directorsChange = val
+    },
+
+    submit () {
+      let changeOfDirectors = null
+
+      const header = {
+        header: { name: 'changeOfDirectors', date: this.currentDate }
+      }
+      const business = {
+        business: {
+          foundingDate: this.entityFoundingDate,
+          identifier: this.entityIncNo,
+          legalName: this.entityName
+        }
+      }
+
+      if (this.isDataChanged('OTCDR')) {
+        changeOfDirectors = {
+          changeOfDirectors: {
+            certifiedBy: 'Full Name',
+            email: 'no_one@never.get',
+            directors: this.$refs.directorsList.getAllDirectors()
+          }
+        }
+      }
+
+      const filingData = {
+        filing: Object.assign(
+          {},
+          header,
+          business,
+          changeOfDirectors
+        )
+      }
+
+      // Temp fix Added to remove cors error while invoking mock endpoint.
+      // To be removed when the API is in place
+      let config = {
+        headers: {
+          'Content-Type': 'text/plain'
+        }
+      }
+
+      axios.post(this.corpNum + '/filings', filingData, config).then(res => {
+        let payRequestId:String = res.data.filing.header.paymentToken
```

Comment:
Should be "let payRequestId: string" - the uppercase types should not be used.

I've been leaving a space after the colon, but that just seems to be the common convention and there is unfortunately no linting around it. I'm in favour of this convention, but I can be convinced otherwise - I do feel strongly about choosing one way and using it consistently (and that's where the linter would help).

> https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

(Actually should be "const" rather than "let", but in this case the next line looks like temporary code that changes the value, so const won't work until the temporary assignment is removed).

Reply:
Will do. I actually copied this from Annual Report so will make the change in both.

You've also alerted me to the hardcoded payRequestId that snuck in (below) - I will take that out it's not supposed to be there.